src/app.scss

```diff
@@ -30,13 +30,17 @@
 }
 
 .files-empty-state,
-.files-view-stack > .pf-v5-c-card {
+.files-view-stack > .pf-v5-c-card,
+.files-view-stack > .files-footer-info {
     grid-column: content;
 }
 
+.files-view-stack > .pf-v5-c-card {
+    overflow: auto;
+}
+
 .files-view-stack > .files-footer-info {
     margin-block-start: 0;
-    grid-column: content;
```

Comment:
As mentioned above, I moved this into a group of all the ones that use the "content" column.